ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -232,6 +232,7 @@
       owner: root
       group: root
       mode: 0644
+    become: 'true'
```

Comment:
Yeah, I had this change on my machine that worked, and didn't check to see it was fixed before submitting.